doc/policy_tutorial/DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,94 @@
+# Setting up the development environment
+
+As was clear from the introduction, APIcast policies are created in the Lua programming language. So we need to setup an environment to do some Lua programming. Also, an actual APIcast server would be very nice to perform some local tests.
+
+Luckily APIcast makes it very easy to setup a development environment for APIcast using Docker and Docker Compose.
+
+### prerequisites
+This means both Docker and Docker compose must be installed.
+
+The version of Docker I currently use is:
+
+    Docker version 19.03.8
+
+Instructions for installing Docker can be found on the Docker [website](https://docs.docker.com/get-docker/).
+
+With Docker compose version:
+
+    docker-compose version 1.25.4
+
+Instructions for installing Docker-compose can also be found on the Docker [website](https://docs.docker.com/compose/install/).
+
+### Setting up the development image
+Now that we have both Docker and Docker-compose installed we an setup the APIcast development image.
+
+Firstly the APIcast git repostitory must be cloned so we can start the development of our policy. Since we are going to base our policy on the latest 3scale release we are switching to the stable branch of APIcast.
+
+```shell
+git clone https://github.com/3scale/apicast.git
+```
+
+when done switch to a stable branch, I am using 3.8
+```shell
+cd apicast/
+git checkout 3.8-stable
+```
+
+To start the APIcast containers using Docker-compose we can use the Make file provided by 3scale. In the APIcast directory simply execute the command:
+```shell
+make development
+```
+
+![make-development](img/make-development.png)
+
+The Docker container starts in the foreground with a bash session. The first thing we need to do inside the container is installing all the dependencies.
+
+This can also be done using a Make command, which again must be issued **inside** the container.
+```shell
+make dependencies
+```
+It will now download and install a plethora of dependencies inside the container.
+
+The output will be very long, but if everything went well you should be greeted with an output that looks something like this:
+
+![make-dependencies](img/make-dependencies.png)
+
+Now as a final verification we can run some APIcast unit tests to see if we are up and running and ready to start the development of our policy.
```

Comment:
Does "as a final verification" refer to a confirmation step related to the installation of the dependencies inside the container? If 'yes`, please reword this sentence to remove the mentioned phrase, because the general policy configuration is still in progress.

Reply:
addressed in #1384